vagrant/Vagrantfile

```diff
@@ -0,0 +1,36 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "generic/ubuntu2004"
+  config.vm.hostname = "script-languages"
+  config.vm.network :forwarded_port, guest: 8888, host: 8888
+  config.vm.network :forwarded_port, guest: 6666, host: 6666
+  config.vm.synced_folder ".", "/home/vagrant/vagrant", type: "rsync", rsync__exclude: ".git/"
+  config.vm.synced_folder "..", "/home/vagrant/script-languages-release", type: "rsync", rsync__exclude: [".git/",".build_output"], rsync__args: ["--verbose", "--archive", "-z", "--links", "--delete"], rsync__verbose: true
+
+  config.vm.provider :libvirt do |v|
+    v.qemu_use_session = true
+    v.cpus = 2
+    v.memory = 6000
+    v.machine_virtual_size = 100
+    v.random_hostname = true
+    v.default_prefix = config.vm.hostname + "_"
+  end
+
+  config.trigger.before [:provision] do |trigger|
+    trigger.run = { inline: "vagrant rsync"}
+  end
+
+  #config.vm.provision "shell" do |s|
+  #   s.path="provisioning/resize-disk.sh"
+  #end
```

Comment:
Was this only a test? Can it be removed? If not, add a comment please.

Reply:
ah, this actually needs uncommented